Modelica/package.mo

```diff
@@ -9079,12 +9079,13 @@ end UsersGuide;
 
 annotation (
 preferredView="info",
-version="4.0.0",
-versionDate="2020-06-04",
-dateModified = "2020-06-04 11:00:00Z",
+version="4.1.0",
+versionDate="2024-03-15",
+dateModified = "2020-01-01 11:00:00Z",
```

Comment:
Was it intentional to push the `dateModified` _backwards_ in time, to January 2020?